cfme/services/myservice/ssui.py

```diff
@@ -225,6 +241,20 @@ def launch_vm_console(self, catalog_item):
     return vm_obj
 
 
+@MyService.retire.external_implementation_for(ViaSSUI)
+def retire(self):
+    view = navigate_to(self, 'Retire')
+    view.retire.click()
+    view = self.create_view(MyServicesView)
+    wait_for(
+        lambda: view.is_displayed, delay=15, num_sec=300,
```

Comment:
**Optional**
Consider dropping this delay to 3s so we're not waiting so long to refresh

Reply:
Done